plugins/autoid/plugin.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
 (function () {
+  var ALPHABET = '23456789abdegjkmnpqrvwxyz';
```

Comment:
I know you link the blog post below, but maybe put a comment in the code explaining why there appear to be letters missing?